package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "foundation-sites",
   "version": "6.4.3",
-  "main": "dist/js/npm.js",
+  "main": "dist/js/foundation.js",
```

Comment:
As you said, reverting to `dist/js/foundation.js` would break a lot of builds. But moving to ES6 as default was definitely not a good move. We're in a complicated position here, with both es6 and non-es6 users expecting to be able to import Foundation with the main.

We cannot accept this PR as it is, but I'll investigate around `jsnext:main`.
Thank you for your contribution 👍 

In your case, what you can do it to import `foundation-sites/dist/js/foundation`
See: https://github.com/zurb/foundation-sites/pull/10782/files